src/tools/user-agent-explainer/user-agent-result-cards.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div v-if="userAgentInfo">
+    <n-grid :x-gap="12" :y-gap="8" cols="1 s:2" responsive="screen">
+      <n-gi v-for="{ heading, icon, content } in sections" :key="heading">
+        <n-card style="height: 100%">
+          <n-page-header>
+            <template #title>
+              {{ heading }}
+            </template>
+            <template v-if="icon" #avatar>
+              <n-icon size="30" :component="icon" :depth="3" />
+            </template>
+          </n-page-header>
+
+          <br />
+
+          <n-space>
+            <span v-for="{ label, getValue } in content" :key="label">
+              <n-tooltip trigger="hover">
+                <template #trigger>
+                  <n-tag v-if="getValue(userAgentInfo)" type="success" size="large" round>
```

Comment:
I find Tags more prettier without border ^^

```suggestion
                  <n-tag v-if="getValue(userAgentInfo)" type="success" size="large" round bordered="false">
```

Reply:
Fixed